app/views/directives/add-config-to-application.html

```diff
@@ -31,22 +31,42 @@
         <legend>Add {{ctrl.apiObject.kind | humanizeKind}} as:</legend>
         <div class="form-group">
           <div class="radio">
-            <div class="add-choice">
-              <label>
-                <input type="radio" ng-model="ctrl.addType" value="env" ng-disabled="ctrl.disableInputs">
-                Environment variables
-              </label>
+            <label class="add-choice" for="env">
+              <input id="env" type="radio" ng-model="ctrl.addType" value="env" ng-disabled="ctrl.disableInputs">
```

Comment:
Instead of having the `ctrl.disableInputs` check on every input, we could wrap the form in  a `<fieldset>` and add the check once there.